src/components/artwork_filter/headline.tsx

```diff
@@ -77,4 +84,14 @@ const StyledHeadline = styled(Headline)`
   font-size: 2em;
 `
 
-export default StyledHeadline
+export default Relay.createContainer(StyledHeadline, {
+  fragments: {
+    facet: () => Relay.QL`
+      fragment on ArtworkFilterFacet {
+        ... on ArtworkFilterTag {
```

Comment:
@mzikherman here is where we are pulling in that metaphysics query on the relay side.